app/src/main/java/com/pajato/android/gamechat/chat/fragment/SelectForInviteFragment.java

```diff
@@ -142,6 +142,48 @@
         return menu;
     }
 
+    /** When a group is selected, find it's common room and insure that it is also selected in
```

Comment:
This comment would hugely benefit by shortening.  "insure" is all about money.  "ensure" is all about determinism and certainty. What works for me is to write a concise doc comment (javadoc), generally a one-liner and to then expand or enhance that comment using in-line comments (//).